automatic/ruby.install/update.ps1

```diff
@@ -0,0 +1,64 @@
+import-module au
+
+$releases = 'https://rubyinstaller.org/downloads/archives/'
+
+function global:au_SearchReplace {
+   @{
+        ".\tools\chocolateyInstall.ps1" = @{
+            "(?i)(^\s*packageName\s*=\s*)('.*')"  = "`$1'$($Latest.PackageName)'"
+            "(?i)(^\s*fileType\s*=\s*)('.*')"     = "`$1'$($Latest.FileType)'"
+        }
```

Comment:
In addition to the requested change for the files in the `chocolateyInstall.ps1` script, please add the replacements:

```suggestion
          "(?i)(^\s*file\s*=\s*`"[$]toolsPath\\).*" = "`${1}$($Latest.FileName32)`""
          "(?i)(^\s*file64\s*=\s*`"[$]toolsPath\\).*" = "`${1}$($Latest.FileName64)`""
        }
```